src/Parser/AST/Statement.php

```diff
@@ -30,4 +32,6 @@ public function __construct(int $sourceLine)
 
         $this->sourceLine = $sourceLine;
     }
+
+    abstract public function getSubNodeNames(): array;
```

Comment:
This breaks the `Helmich\TypoScriptParser\Tests\Unit\Parser\AST\StatementTest`, which generates an anonymous subclass of `Statement` without implementing `getSubNodeNames`.

Reply:
done